rpm/archivematica/package.spec

```diff
@@ -0,0 +1,342 @@
+# Globals
+%global __os_install_post %(echo '%{__os_install_post}' | sed -e 's!/usr/lib[^[:space:]]*/brp-python-bytecompile[[:space:]].*$!!g')
+
+### Packages
+Name: archivematica
+Version: 1.4.1
+Release: 1
+Summary: Archivematica 
+Group: Application/System
+License: AGPLv3
+Source0: https://github.com/artefactual/archivematica/
+BuildRequires: git, gcc, openssl-devel, python-virtualenv, python-pip, mariadb-devel, libxslt-devel, python-devel
+Requires: archivematica-common
+AutoReq: No
+AutoProv: No
+
+%description
+Archivematica dashboard and MCP server
+
+%package mcp-client
+Summary: archivematica-mcp-client
+Requires: archivematica-common, sudo, tesseract, p7zip, ImageMagick, ghostscript, perl-Image-ExifTool, inkscape, python-unidecode, clamav-server clamav-data clamav-update clamav-filesystem clamav clamav-scanner-systemd clamav-devel clamav-lib clamav-server-systemd
+AutoReq: No
+AutoProv: No
+
+%description mcp-client
+MCP Client for Archivematica
+
+%package common
+Summary: Archivematica common files
+
+%description common
+Common files for Archivematica
+
+### Files 
+%files
+/usr/share/archivematica/
+/usr/lib/systemd/system/archivematica-dashboard.service
+/usr/lib/python2.7/archivematica/dashboard
+%config /etc/sysconfig/archivematica-dashboard
+
+# MCP Server
+/usr/lib/archivematica/MCPServer
+/usr/lib/systemd/system/archivematica-mcp-server.service
+%config /etc/archivematica/MCPServer/serverConfig.conf
+%config /etc/sysconfig/archivematica-mcp-server
+
+# MCP Client
+%files mcp-client
+/usr/lib/python2.7/archivematica/MCPClient/
+/usr/lib/archivematica/MCPClient/
+/usr/lib/systemd/system/archivematica-mcp-client.service
+%config /etc/archivematica/MCPClient/archivematicaClientModules
+%config /etc/archivematica/MCPClient/clientConfig.conf
+%config /etc/sysconfig/archivematica-mcp-client
+# Common
+%files common
+/usr/lib/archivematica/archivematicaCommon/
+/var/archivematica/sharedDirectory
+/usr/share/archivematica-common
+%config /etc/archivematica/archivematicaCommon/dbsettings
+
+### Install
+%install
+
+# Dashboard
+mkdir -p %{buildroot}/usr/lib/python2.7/archivematica/dashboard
+virtualenv /usr/lib/python2.7/archivematica/dashboard
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install --upgrade pip
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install -r %{_sourcedir}/%{name}/src/archivematicaCommon/requirements/production.txt
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install -r %{_sourcedir}/%{name}/src/dashboard/src/requirements/production.txt
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install mysql
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install gunicorn
+# ??
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install jsonfield
+# Needed by MCP Server:
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install pyinotify
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install lxml
+/usr/lib/python2.7/archivematica/dashboard/bin/pip install gearman
```

Comment:
In `qa/1.x` you shouldn't need any of ^, all the deps are included properly in the pip `requirements`.